### ISSUE TITLE
Support datetime.timedelta field

### ISSUE THREAD

**Issue by kozickikarol** [aminalaee/sqladmin#327]
### Checklist

- [X] There are no similar issues or pull requests for this yet.

### Is your feature related to a problem? Please describe.

Currently creating/editing instance with timedelta is not possible. sqladmin renders DateTime picker (which is wrong) and after saving it throws 500.

```
site-packages/wtforms/fields/datetime.py", line 36, in _value
return self.data and self.data.strftime(self.format[0]) or ""
AttributeError: 'datetime.timedelta' object has no attribute 'strftime'
```
Viewing timedelta is possible (it renders duration in humanized format e.g. 1 day, 0:00:00 which is quite good).

### Describe the solution you would like.

MVP: format 00:00:00  (hours:minutes:seconds)
Later e.g. sth like that: 
![image](https://user-images.githubusercontent.com/8099900/191109373-7374a2b5-33e6-4542-bbbb-36f393b5c3f5.png)

**Comment by aminalaee** [comment]
@kozickikarol You mean support for the `Interval` field, right? The `Interval` field will use `datetime.timedelta`.

**Comment by Giteq** [comment]
Any update on this?

**Comment by aminalaee** [comment]
@Giteq Just added this. This will be available in version 0.10.0
